### PR TITLE
[WNMGDS-911] Update type to use agnostic styles

### PR DESCRIPTION
**Pull request by line47**
- Adding agnostic class names to SCSS files
- Adding agnostic type variables

### CODE REVIEW

File: src/styles/base/_override.typography.scss [hunk 1]
```diff
@@ -6,22 +6,37 @@
 .ds-h3,
 .ds-h4,
 .ds-h5,
-.ds-h6 {
+.ds-h6,
+.ds-text-heading--5xl,
```
Comment (line47) [inline src/styles/base/_override.typography.scss:10]: Added support for new classes https://cmsgov.github.io/mgov-design-system/styles/typography/

File: src/styles/settings/_variables.type.scss [hunk 2]
```diff
@@ -1,12 +1,30 @@
+$font-sans: "Rubik", sans-serif;
 $rubik-font: "Rubik", sans-serif;
 $montserrat-font: "Montserrat", sans-serif;
 
-$font-sans: "Rubik";
+$base-font-size: 16px !default;
+
+$font-size-5xl: 50px !default;
```
Comment (line47) [inline src/styles/settings/_variables.type.scss:7]: Added new font size variables

**Review by babsmiles — APPROVED** [review]
This looks great! Thank you!

**Comment by line47** [comment]
@babsmiles it looks like [CMS-MCT/coverage-tools-frontend ](https://github.cms.gov/CMS-MCT/coverage-tools-frontend) is using `mct-p1` styles. Could we add a task to their backlog for updating to use the new agnostic classes ?

**Review by nichia — APPROVED** [review]
:rocket:

**Pull request merged**